Controller/PasswordResetController.php

```diff
@@ -35,10 +35,10 @@ public function requestAction(Request $request)
 
                 return $this->redirectToRoute('multi_user_login');
             } catch (EntityNotFoundException $exception) {
-                $errorMessage = $this->get('translator')->trans(
+                $errorMessage = ucfirst($this->get('translator')->trans(
```

Comment:
had je dat eerder niet aangepast? of moet dat hier ook niet aangepast worden dat dat in de validations messages zit?